### PR TITLE
OCPQE-19090: Regenerating certs support for gcp workload identity

### PR DESCRIPTION
**Pull request by wangke19**
To add regenerating certs support for the GCP cluster with gcp workload identity, related Jira issue https://issues.redhat.com/browse/OCPQE-19090.
Add test jobs for 4.12 - 4.16 releases.

### CODE REVIEW

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14

**Comment by wangke19** [comment]
/pj-rehearse periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-gcp-ipi-proxy-etcd-encryption-f28

**Comment by wangke19** [comment]
/pj-rehearse  periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-gcp-ipi-workload-identity-rt-fips-regen-cert-f28

**Comment by wangke19** [comment]
Executed three jobs to verify the PR fix, 
One normal job of gcp test profile:  periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-gcp-ipi-proxy-etcd-encryption-f28
Two jobs of regenerating certs support for the GCP cluster with gcp workload identity: 
periodic-ci-openshift-openshift-tests-private-release-4.14-amd64-nightly-gcp-ipi-oidc-rt-fips-regen-cert-f14
periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-nightly-gcp-ipi-workload-identity-rt-fips-regen-cert-f28

**Comment by xingxingxia** [comment]
Well received. Will review but stay tuned, may have some delay from my side due to an urgent feature I talked to you.

File: ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-commands.sh [hunk 1]
```diff
@@ -154,31 +162,30 @@ oc adm restart-kubelet nodes --all --directive=RemoveKubeletKubeconfig
...
-
 # now that trust is distributed, create new certificates using new roots
 oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver-operator secrets node-system-admin-client
-oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver secrets check-endpoints-client-cert-key control-plane-node-admin-client-cert-key  external-loadbalancer-serving-certkey internal-loadbalancer-serving-certkey kubelet-client localhost-recovery-serving-certkey localhost-serving-cert-certkey service-network-serving-certkey
+oc adm ocp-certificates regenerate-leaf -n openshift-kube-apiserver secrets check-endpoints-client-cert-key control-plane-node-admin-client-cert-key external-loadbalancer-serving-certkey internal-loadbalancer-serving-certkey kubelet-client localhost-recovery-serving-certkey localhost-serving-cert-certkey service-network-serving-certkey
 oc adm wait-for-stable-cluster
-
 # create new admin.kubeconfig
-oc config new-admin-kubeconfig > "${SHARED_DIR}/admin.kubeconfig"
+oc config new-admin-kubeconfig >"${SHARED_DIR}/admin.kubeconfig"
```
Comment (liangxia) [inline ci-operator/step-registry/openshift/e2e/test/qe/regenerate-cluster-cert/openshift-e2e-test-qe-regenerate-cluster-cert-commands.sh:170]: IMO, this kind of changes should be reverted, as the former one are easier to read.

**Pull request merged**